content/en/blog/_posts/2023-12-13-kubernetes-1.29.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: blog
+title: 'Kubernetes 1.29: Mandala'
```

Comment:
@katcosgrove previous release announcements **didn't** include the “v” in the title. For the title, I'd stick with the convention we have (we could change for a future release though).

Reply:
@sftim  They have since v1.25, which might actually be my fault since I was the comms lead for v1.25 lol. I don't have a strong preference here, though!

Reply:
@sftim  and @katcosgrove I did a research for the release blog convention: `1.18, 1.19, 1.20, 1.21, 1.22, 1.23, 1.24, v1.25, v1.26, v1.27, v1.28`